apps/web/app/actions.ts

```
@@ -2,118 +2,200 @@
+		})
+
+		if (error) {
+			return errorHandler.handleAuthError(error, 'sign_in')
+		}
+
+		return {
+			success: true,
+			message: 'Successfully signed in',
+			redirect: '/dashboard',
+		}
+	} catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)